crates/rust-analyzer/src/reload.rs

```diff
@@ -83,8 +83,10 @@ impl GlobalState {
             status.health = lsp_ext::Health::Warning;
             status.message = Some("Workspace reload required".to_string())
         }
-
-        if let Some(error) = self.fetch_workspace_error() {
+        if self.fetch_workspaces_queue.op_in_progress() {
+            status.health = lsp_ext::Health::Warning;
```

Comment:
Hm `Warning` doesn't sit quite right with me, would it make sense to keep health as Ok here?

Reply:
Hm, I hesitated for a long time because it's an unknown status.
For example, I write the cargo toml like this:
```
[dependencies]
11
```
and then, I delete the second '1', actually, there's still an error, we do not know the fetching result at that moment.

Reply:
r-a doesn't really know about `Cargo.toml` errors at all, and when `op_in_progress` is true that doesn't mean there was a problem with loading, it's just that we are currently loading. Maybe it would make sense to a fourth variant to the `Health` enum, `Health::Info` or something like that?

Reply:
totally agree!

Reply:
I think OK is same as Info, so Ok looks good to me. No need changes to lsp_ext. And I accidentally closed this pr 😭, I will submit new one based on the latest master branch. @Veykril